frontend/src/components/layouts/components/Sider.tsx

```diff
@@ -0,0 +1,30 @@
+import styled from "styled-components";
+import type { MenuProps } from 'antd';
+import { Menu as AMenu } from 'antd';
+
+const Sider =styled(AMenu)`
+
+margin: 0;
+width: 260px;
+background-color: #232a44;
+top: 0px;
```

Comment:
To też nie jest potrzebne, mamy to już załatwione przez samego antd. On pozwala ułożyć layout. Swoją drogą wcześniej już było dobrze i jedyne co musisz zmienić to tylko kolor tła i scrollbara. Cała reszta styli jest do wywalenia.